install/docker-compose.base.yml

```diff
@@ -24,3 +24,4 @@ services:
       - FOSSOLOGY_DB_HOST=fossology-db
       - FOSSOLOGY_DB_NAME=fossology
       - FOSSOLOGY_DB_USER=fossy
+    user: fossy
```

Comment:
Why is this visible in the diff here? These changes are already merged. Maybe you should rebase on master or cleanup your branch.